### DISCUSSION TITLE
Adding custom link to Jellyfin Web.

### DISCUSSION THREAD

**Discussion by fattmatt1972** [jellyfin/jellyfin-web#5912] [General]
Hi all.

I am a Jellyfin Server user and I am dabbling with doing a few edits to the webpage using the repo.

I am trying to get an extra link on the top next to the 'Favorites" tab, I have managed to get it to show using the js home page settings, this is the code I have atm for the link I want, under the 'get tabs' section.

getTabs() {
return [{
name: globalize.translate('Home')
}, {
name: globalize.translate('Favorites')
}, {
name: globalize.translate('Requests'),
href: ('https://xxx.xxx.xyz')
}
];
}

The link shows but when I click it, it resolves in the address bar after the current Jellyfin address.

It obviously wants to open a local page and I don't know how to make it open externally.

I have tried adding target: ('_blank'), rel: ('noopener noreferrer') to the end of my link, like this...

name: globalize.translate('Requests'),
href: ('[https://xxx.xxx.xyz'), target: ('_blank'), rel: ('noopener noreferrer')

but when I look at the code on the site it drops the target/rel part, but it does see it as an 'emby-link-tab' and does fill the right address, just not in a new tab.

All the references I can find relate to HTML or other js code that doesn't work, I am very new to this and am not really sure how to proceed.

Thanks in advance,

Matt.

**Comment by thornbill** [comment]
We don't support this type of customization.
